front-end/e-learning/src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Login from './Login';
```

Comment:
put a space to group imports

```suggestion
import React from 'react';

import Login from './Login';
```